src/main/resources/application.yml

```diff
@@ -28,38 +28,6 @@ spring:
     console:
       enabled: true
 
-  server:
```

Comment:
server etc. sections below this should have been at the root rather than under `spring:`